docker/dev/docker-compose.yml

```diff
@@ -70,9 +70,9 @@ services:
       NODE_ENV: development
       PORT: 3004
       DATABASE_URL: postgres://boutique_backend:boutique_backend@postgres/boutique_backend
-      PRIVATE_KEY: ${PRIVATE_KEY}
-      KEY_ID: ${KEY_ID}
-      PAYMENT_POINTER: ${PAYMENT_POINTER}
+      PRIVATE_KEY: LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0tLS0tCk1DNENBUUF3QlFZREsyVndCQ0lFSUI0YzgyOVlSMDZCTUhtQmpIVTNrOHZmMEZhVEFvOHNCTzIrRFZwY1lPSk0KLS0tLS1FTkQgUFJJVkFURSBLRVktLS0tLQ==
```

Comment:
Boutiques is not working, so I am guessing a key is not correct. Looking back on previous implementation locally with using keys from environment variables, I have the Private Key with only one '=' at the very end. You have '=='.
I think this is the new line added into encoding. Please try with one '='.